data/schedule.yaml

```diff
@@ -9,12 +9,13 @@ locations:
         by: Bloomberg
         time: '08:30'
         description: >
-          Help yourself to a bite to eat and a cup of coffee, courtesy of Bloomberg!
+          Help yourself to a bite to eat and a cup of coffee, courtesy of
+          Bloomberg!
       - name: Introduction
         by: The SISTEM Team
         time: '09:00'
         description: >
-          A welcoming speech to begin the day from some of the organisers
+          A welcoming speech to begin the day with from some of the organisers
```

Comment:
`with from`

Reply:
`with from` is correct

Reply:
the alternative is `begin the day, from`

Reply:
Brain slow, need coffee. Woops.